viewer/viewer/templates/viewer/help.html

```diff
@@ -46,8 +46,15 @@ <h2>Components</h2>
     Search by
     <a href="{% url 'components' %}">class names</a>
     of our
-    <a href="https://cfpb.github.io/design-system/">design components</a>.
-    Partial searches also work: searching for "notification" will return pages
+    <a href="https://cfpb.github.io/design-system/">design components</a>
+    <svg
+      class="cf-icon-svg"
+      viewBox="0 0 14 19"
+      xmlns="http://www.w3.org/2000/svg">
+      <path
+        d="M13.017 3.622v4.6a.554.554 0 0 1-1.108 0V4.96L9.747 7.122a1.65 1.65 0 0 1 .13.646v5.57A1.664 1.664 0 0 1 8.215 15h-5.57a1.664 1.664 0 0 1-1.662-1.663v-5.57a1.664 1.664 0 0 1 1.662-1.662h5.57A1.654 1.654 0 0 1 9 6.302l2.126-2.126H7.863a.554.554 0 1 1 0-1.108h4.6a.554.554 0 0 1 .554.554zM8.77 8.1l-2.844 2.844a.554.554 0 0 1-.784-.783l2.947-2.948H2.645a.555.555 0 0 0-.554.555v5.57a.555.555 0 0 0 .554.553h5.57a.555.555 0 0 0 .554-.554z"></path>
+    </svg>
+    . Partial searches also work: searching for "notification" will return pages
```

Comment:
Technically I think this needs to be what is documented [here](https://cfpb.github.io/design-system/components/links#links-with-icons-1):

```
<a class="a-link a-link__icon" href="...">
  <span class="a-link_text">...</span>
  <svg>...</svg>
</a>.
```

so that the icon itself is also part of the link. See [another example](https://github.com/cfpb/crawsqueal/blob/0085d0d909c90f06747f708a2a312157954f4e45/viewer/viewer/templates/viewer/base.html#L131-L141) in this repo.